Multisig/UI/Safe Management/Create Safe/SafeCreatingViewController.swift

```diff
@@ -66,13 +59,17 @@ class SafeCreatingViewController: UIViewController {
 
     @objc func accountCreated() {
         infoView1.set(status: .success)
+        // It is not defined, what this "owner secured" exactly means. Thats why we check this box after a short time automatically
+        Timer.scheduledTimer(withTimeInterval: 1, repeats: false) {_ in
+            NotificationCenter.default.post(name: .safeAccountOwnerSecured, object: nil)
+        }
```

Comment:
There was a todo comment above, so maybe this wasn't fully implemented and left for completion?

Reply:
Otherwise, since the whole block was deleted, maybe then this is a dead code now?